ThrowIT/Utility.h

```diff
@@ -35,21 +35,26 @@
 #define PARTYCLASS @"Party"
 #define THROWERCLASS @"Thrower"
 #define ATTENDANCECLASS @"Attendance"
+#define USERCLASS @"_User"
 
 #pragma mark - Parse Keys
 #define USER @"user"
 #define USERISTHROWERKEY @"isThrower"
+#define USERPROFILEPHOTOKEY @"profilePhoto"
+#define USERUSERNAMEKEY @"username"
+#define PARTIESATTENDEDKEY @"partiesAttended"
 
 #define THROWERKEY @"thrower"
 #define VERIFIEDKEY @"verified"
 #define THROWERNAMEKEY @"throwerName"
+#define THROWERRATING @"throwerRating"
 
 #define PARTYKEY @"party"
 #define PARTYTHROWERKEY @"partyThrower"
 
 #define ATTENDANCETYPEKEY @"attendanceType"
-#define NOTGOING @"Not going"
-#define GOING @"Going"
+#define NOTGOING @"Pass"
```

Comment:
it's good practice to make your pull requests/ changes in smaller commits so it's easier to to track them as you continue to develop. for example, you can have a commit to just update the string value for the defines here or adding a UI component like the ProfileView (may contain a view, view controller, and model)

Reply:
Just seeing this, will be more aware to do so